app/assets/javascripts/modules/hide-other-links.js

```diff
@@ -0,0 +1,87 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function HideOtherLinks (module) {
+    this.module = module
+    this.anchors = this.module.querySelectorAll('a')
+    this.childNodes = this.module.childNodes
+    this.hiddenElementContainer = this.createHiddenElementContainer()
+    this.shownElements = []
+    this.hiddenElements = []
+    this.showLink = document.createElement('a')
+  }
+
+  HideOtherLinks.prototype.init = function () {
+    if (this.anchors.length <= 2 && this.module.textContent.length <= 100) {
+      return
+    }
+
+    this.sortChildNodes()
+
+    if (!this.hiddenElements.length) {
+      return
+    }
+
+    this.hideElements()
+    this.createShowLink()
+
+    this.module.setAttribute('aria-live', 'polite')
+  }
+
+  HideOtherLinks.prototype.sortChildNodes = function () {
+    var linksCount = 0
+
+    this.childNodes.forEach(function (node) {
+      if (linksCount >= 1) {
+        this.hiddenElements.push(node)
+      } else {
+        this.shownElements.push(node)
+      }
+
+      if (node.nodeName.toLowerCase() === 'a') {
+        linksCount += 1
+      }
+    }, this)
+  }
+
+  HideOtherLinks.prototype.createShowLink = function () {
+    var hiddenCount = this.hiddenElementContainer.children.length
+    var linkText = `+ ${hiddenCount} other${hiddenCount > 1 ? 's' : ''}`
```

Comment:
Sorry to nitpick but template strings aren't supported by IE 11 so you would need to use string concatenation here.

Reply:
👍 I'll update that thanks! Does the rest of this file look okay? I haven't written JS in this style before

Reply:
Change made